README.md

```diff
@@ -1,29 +1,20 @@
-# decentralized-renderer-react-template
+# generic-template
```

Comment:
Maybe a short description to explain what this does and who it can help?

Reply:
ok